libraries/ui-shared/icons/ShareIcon.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '@shopgate/pwa-common/components/Icon';
+
+/**
+ * The filter icon component. SVG is grabbed from config
+ * @param {Object} props The icon component properties.
+ * @returns {JSX}
+ */
+const ShareIcon = (props) => {
```

Comment:
I'm sorry but I think we should not mix it. Let's keep two `ShareIcon`, one in ui-ios and one in ui-gmd so it's obvious and easy to use.

Reply:
.

Reply:
Made to ShareIcon files.